mail-mta/sendmail/sendmail-8.18.1.15.ebuild

```
@@ -289,4 +306,11 @@ pkg_postinst() {
 		elog "configuration file:"
 		elog "\tFEATURE(\`local_procmail',\`/usr/bin/maildrop',\`maildrop -d $u')dnl"
 	fi
+
+	ewarn "This version has enabled experimental code. Please read the file"
+	ewarn ""
+	ewarn "\t/usr/share/doc/${P}/SNAPSHOT_NOTES"
```

```suggestion
	ewarn "\t/usr/share/doc/${PF}/SNAPSHOT_NOTES"
```

(If the package has -r1..)